sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -123,12 +123,21 @@
 
   public static final String NCLOC_KEY = "ncloc";
   public static final Metric<Integer> NCLOC = new Metric.Builder(NCLOC_KEY, "Lines of Code", Metric.ValueType.INT)
-    .setDescription("Non Commenting Lines of Code")
+    .setDescription("Non commenting lines of code")
     .setDirection(Metric.DIRECTION_WORST)
     .setQualitative(false)
     .setDomain(DOMAIN_SIZE)
     .create();
 
+  public static final String NEW_NCLOC_KEY = "new_ncloc";
+  public static final Metric<Integer> NEW_NCLOC = new Metric.Builder(NEW_NCLOC_KEY, "New Lines of Code", Metric.ValueType.INT)
+    .setDescription("New non commenting lines of code")
+    .setDirection(Metric.DIRECTION_WORST)
+    .setQualitative(false)
+    .setDomain(DOMAIN_SIZE)
+    .setHidden(true)
```

Comment:
because we're not interested to display it in the UI, it doesn't help to handle the leak period